docker-compose.dev.yaml

```diff
@@ -2,57 +2,41 @@ version: "3.7"
 
 services:
   init:
```

Comment:
all places where we mount directories need to be overridden here now since prod version uses named mounts and dev version mounts  directories.